packages/server/src/api/controllers/market-data.ts

```diff
@@ -74,22 +74,23 @@ async function getPoolIndicators(
     return { marketData, indicators: poolIndicators };
 }
 
+const cacheConfig = { maxAge: 15, sMaxAge: 300, public: true, mustRevalidate: true };
```

Comment:
users should check for new data every 15s, cdn will hold for 5 min before revalidating with servers